content/api/rest-api.mdx

```diff
@@ -0,0 +1,1642 @@
+- limit := _100_ The maximum number of records to return. A limit greater than 1,000 is invalid.
+
+##### Options
+
+- Content-Type := not applicable
+- Accept := `application/json` by default, or `application/x-msgpack`, `text/html`
+- Auth required := yes ([basic](#basic-authentication) or [token](#token-authentication))
+
+##### Returns
+
+A successful request returns a [paginated response](#pagination) with an array containing the members that are currently present on the given channel. If there are no members present, an empty collection is returned.
+
+```json
+[{
+  id: <a unique member identifier generated by Ably>,
+  clientId: <member client id provided by the client>,
+  connectionId: <a unique connection id generated by Ably>
+  timestamp: <message timestamp in ms since epoch>
+  action: <presence state>,
+  data: <optional clientData provided by the client>
+}]
+```
+
+### Retrieve presence state history for a channel {#presence-history}
+
+#### GET main.realtime.ably.net/channels/`<channelId>`/presence/history
+
+Obtain the history of presence messages for a channel.
+
+Example request:
+
+```bash
+curl https://main.realtime.ably.net/channels/rest-example/presence/history \
+ -u "{{API_KEY}}"
+```
+
+##### Parameters
+
+- start := _beginning of time_ The start of the query interval as a time in milliseconds since the epoch. A message qualifies as a member of the result set if it was received at or after this time.
+- end := _now_ The end of the query interval as a time in milliseconds since the epoch. A message qualifies as a member of the result set if it was received at or before this time.
+- limit := _100_ The maximum number of records to return. A limit greater than 1,000 is invalid.
+- direction := _backwards_ The direction of this query. The direction determines the order of the returned result array, but also determines which end of the query interval is the start point for the search.  For example, a forwards query uses `start` as the start point, whereas a backwards query uses `end` as the start point.
+
+##### Options
+
+- Content-Type := not applicable
+- Accept := `application/json` by default, or `application/x-msgpack`, `text/html`
+- Auth required := yes ([basic](#basic-authentication) or [token](#token-authentication))
+
+##### Returns
+
+A successful request returns a [paginated response](#pagination) with an array containing the members that are currently present on the given channel. If there are no members present, an empty collection is returned.
+
+```json
+[{
+  id: <a unique member identifier generated by Ably>,
+  clientId: <member client id provided by the client>,
+  connectionId: <a unique connection id generated by Ably>
+  timestamp: <message timestamp in ms since epoch>
+  action: <presence state>,
+  data: <optional clientData provided by the client>
+}]
+```
+
+### Retrieve metadata for a channel {#metadata-rest}
+
+<%= partial partial_version('shared/_channel_metadata') %>
```

Comment:
Partials need to be pulled into the file instead of referenced. (And sadly then converted)